cypress/integration/shared.js

```diff
@@ -451,6 +451,12 @@ export const saveClusterDetails = (cy) => {
   // click the 'save' button in order to save changes in the cluster info
   cy.get('button[name="save"]', { timeout: VALIDATE_CHANGES_TIMEOUT }).should('be.enabled');
   cy.get('button[name="save"]').click();
+  cy.wait(2 * 1000);
+};
+
+export const enableRoute53 = (cy) => {
+  cy.get('#form-input-useRedHatDnsService-field').click();
+  cy.get('#form-input-baseDnsDomain-field').contains('route53');
```

Comment:
Maybe rather than checking for route53 hard coded you can make a GET request to /domains and compare to the response in case route53 is changed in the future

Reply:
Refactored the check